### ISSUE TITLE
SEA server - can't connect after accepting game

### ISSUE THREAD

**Issue by anemonemone** [jeffhill/Dota2#741]
### Description

game fails to load hero selection screen after accepting match. after clicking the "reconnect" button, the game attempts to load the hero selection screen but fails to do so. this has happened four times back to back. each time it counts as failure to accept and i am unable to search for a game for an increasing amount of time.

**Comment by aoisensi** [comment]
Try `-tcp` option when launching **steam.exe**.

**Comment by hmzakhalid** [comment]
Yeah lol I just got 30 Mins cuz of it. I thought it was my internet but guess not
